desktop/desktop.scss

```diff
@@ -1,11 +1,5 @@
-.link-bottom-line {
-  font-size: unset;
-}
-
-.topic-list-item .discourse-tags {
-  display: block;
-  font-size: var(--font-down-2);
-  clear: both;
+.link-bottom-line .badge-wrapper {
```

Comment:
.badge-wrapper was replaced with badge-category__wrapper in https://github.com/discourse/discourse/commit/797da5870be1e651f68d32ec47374a057c878625 (moving to single category badge style)

Reply:
Thanks - fixed